clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -128,15 +136,39 @@ func (p *podStartupLatencyMeasurement) start(c clientset.Interface) error {
 				return c.CoreV1().Pods(p.selector.Namespace).Watch(context.TODO(), options)
 			},
 		},
-		p.checkPod,
+		p.addEvent,
 	)
+	go p.processEvents()
 	return informer.StartAndSync(i, p.stopCh, informerSyncTimeout)
 }
 
+func (p *podStartupLatencyMeasurement) addEvent(_, obj interface{}) {
+	event := &eventData{obj: obj, recvTime: time.Now()}
+	p.eventQueue.Add(event)
```

Comment:
nit: add a comment (on the eventQueue definition) that it can potentially grow indefinitely, because we put changes (which isn't the recommended patern, but it's what we really need here).

Reply:
Done.